### PR TITLE
Create phil-uk-file.md

### PR DESCRIPTION
**Pull request by Phil-Clearstream**
### Description of Change

- creating the UK file

### Test results

- Not tested

### Who should know about the change?

/cc: @githubteacher

### CODE REVIEW

File: phil-uk-file.md [hunk 1]
```diff
@@ -0,0 +1,3 @@
+London,
```
Comment (Phil-Clearstream) [inline phil-uk-file.md:1]: reviewed ok

**Review by Phil-Clearstream — COMMENTED** [review]
ok

**Review by pm700 — COMMENTED** [review]
looks good

File: phil-uk-file.md (hunk 1 above)
Comment (jq326) [inline phil-uk-file.md:1]: Where can I eat?

**Review by pm700 — APPROVED** [review]

**Pull request merged**